### PR TITLE
Add new watchfaces. New app to pick watchfaces

### LINKED ISSUE #254: Add support for watchfaces from esp32-lvgl-watchface
**Link**
https://github.com/fbiego/esp32-lvgl-watchface

**Modified repo**
https://github.com/Kampi/esp32-lvgl-watchface/tree/ZSWatch_Port

**ToDo**
- [x] Modify Kotlin script to support the newest compiler (replace `XYcase` with `toXYCase` and add label for `continue` in `when` statement)
- [x] Clean up the repository
- [x] Modify the template generator to avoid the usage of additional glue code
- [x] Update README
- [x] Rename color swap macro to match with Zephyr naming scheme
- [x] Add CMakeList generation to the template generator
- [ ] Check the generated code for batteries and check if we can use the `percent` parameter
- [ ] Add Kotlin code for number of notifications
- [ ] Add Kotlin code for environmental sensors
- [x] Move all images to a separate directory (one directory for all watchfaces) to make them work with the external flash memory implementation. Also copy this directory into the right location
- [ ] LLEXT interface for watchfaces (requires USB and Drag & Drop)

**Bugs**
- `face_1083_dial_img_33_203425_group` missing when parsing `dials/X5Pro/1083` (@fbiego can you help here?)

**Notes**
- All in one command: 
```
kotlinc bin2lvgl.kt -include-runtime -d bin2lvgl.jar && java -jar bin2lvgl.jar 24_2_dial.bin && mkdir -p ../ZSWatch/app/src/ui/watchfaces/24_2_dial && cp 24_2_dial/* ../ZSWatch/app/src/ui/watchfaces/24_2_dial/
```

### PR DESCRIPTION
**Pull request by Kampi**
Close #254

### CODE REVIEW

File: app/src/applications/watchface/watchface_app.h [hunk 1]
```diff
@@ -42,7 +42,8 @@ typedef struct watchface_ui_api_t {
     void (*set_ble_connected)(bool connected);
     void (*set_num_notifcations)(int32_t value);
     void (*set_weather)(int8_t temperature, int weather_code);
-    void (*set_date)(int day_of_week, int date);
+    void (*set_shortdate)(int day_of_week, int date);
```
Comment (jakkra) [inline app/src/applications/watchface/watchface_app.h:45]: I'd just edit `set_date` to include all params as `set_fulldate`, no need for a another function for this. Unless you had a reason to do this?

Comment (Kampi) [reply app/src/applications/watchface/watchface_app.h:45]: Basically for legacy reasons. But ofc. I can drop it.

Comment (jakkra) [reply app/src/applications/watchface/watchface_app.h:45]: Drop it :)

Comment (Kampi) [reply app/src/applications/watchface/watchface_app.h]: Changed :)

**Comment by jakkra** [comment]
Note: additional `sudo apt install openjdk-11-jdk` is needed to run script

**Comment by Kampi** [comment]
Looks good so far!

**Pull request merged**